content/resources/lessons/sound-basics/presentation/index.md

```diff
@@ -134,6 +169,19 @@ $$
 -   [mobilebeat.com/audio-bit-depth-and-sample-rate](https://www.mobilebeat.com/audio-bit-depth-and-sample-rate)
 
 {{% /row %}}
+
+>Notes:
+> - An analogue signal is a continuous line
+> - We use sampling or quantization to capture various points along that signal
+> - Can then transform the time-discrete signal with quantization, or a quantized
+>   signal with sampling, into a digital signal.
+> - The sample rate refers to how many samples are captured each second. For
+>   example, a sampling rate of 22050 means that each second, 22050 samples are
+>   captured.
+> - The bit depth refers to how many bits of information are stored for each
+>   sample.
```

Comment:
sample rate and bit depth are both resolution parameters along different axes. Your explanation is accurate but missing an important point: lower bit depth means less accurate quantization, higher means more accurate quantization. Or in other words: bit depth is affects dynamic range.

Reply:
thanks, updated to include this extra detail